data/monitoring/prom-rule.yaml

```diff
@@ -19,3 +19,11 @@ spec:
           for: 5m
           labels:
             severity: warning
+        - alert: CnaoNotReady
```

Comment:
Maybe I am late to the discussion but this feals like the CNAO operator is not ready more than the CNAO CR, I think this have ot be NetworkAddonsConfigNotReady

Reply:
I agree. Done